cmd/generate-release/scripts/install-asm.bash

```diff
@@ -33,7 +33,7 @@ else
     echo "gcloud CLI is already available, skipping installation"
 fi
 
-curl https://storage.googleapis.com/csm-artifacts/asm/asmcli_1.16 >asmcli
+curl https://storage.googleapis.com/csm-artifacts/asm/asmcli_1.17 >asmcli
```

Comment:
I think you also have to bump: https://github.com/google/kf/blob/304fe55816f05a8d706ad0f451dbc6a335ebbee4/pkg/kf/commands/dependencies/dependencies.go#L123 to get the dependency matrix to work